variables.tf

```diff
@@ -1,21 +1,38 @@
-# TEMPLATE: All variables must have a description and should declare their type.
-# TEMPLATE: Set defaults whenever possible but do not set defaults for required properties.
-# TEMPLATE: Declare all variables in this file, sprawling declarations are difficult to identify.
-# TEMPLATE:
-# TEMPLATE: https://www.terraform.io/docs/language/values/variables.html
-# TEMPLATE: https://www.terraform.io/docs/language/expressions/types.html
-# TEMPLATE:
+variable "runner_scope" {
+  type        = string
+  description = "To specify repositories, use the format ':owner/:repo'. For example, 'equinix-labs/terraform-equinix-github-runner'. To specify an organization, use the format ':organization'"
+}
 
-# TEMPLATE: Replace sample variable described below with your own.
-variable "metal_project_id" {
+variable "personal_access_token" {
   type        = string
-  description = "The example project id value defines what will be included in the example resource in main.tf. This example is descriptive."
-  sensitive   = false
+  description = "GitHub PAT (Personal Access Token)"
```

Comment:
Sorry, I think I meant to suggest adding `sensitive` here?

Reply:
It's a workaround. If put sensitive in the variable it hides all stdout of null_resource `wait` https://github.com/equinix-labs/terraform-equinix-github-runner/blob/80ac4268077f1231651cd5f4cf151faac42105d4/main.tf#L37 even the variable is set as an environment variable and never printed in the log. I want that stodout printed in the log so it gives information of the wait process. However, in the null_resource  `delete_script` it needs to be set in the triggers section, because provisioners with `when=destroy` meta-argument can only access `self` variables, and there's no way to supress triggers to be printed in the log other than using a sensitive variable

Reply:
Could we reverse the polarity on that? I think keeping the token sensitive by default would reduce the risk of leaking it in terraform output.  In that case, it looks like we could declare the variable as `sensitive` and then use `RUNNER_CFG_PAT = nonsensitive(var.personal_access_token)`.
https://github.com/hashicorp/terraform/issues/27154#issuecomment-1070983147

Reply:
Sure! It makes total sense, I didn't know about `nonsensitive` function

Reply:
I added that suggested change and worked well for both create and destroy process

![image](https://github.com/equinix-labs/terraform-equinix-github-runner/assets/68897552/09466a40-0ae1-4f2d-acb3-3b8158f2f2c7)
![image](https://github.com/equinix-labs/terraform-equinix-github-runner/assets/68897552/80c24a8f-94e2-424f-9b84-b6543e1e65cb)